README.md

```diff
@@ -25,42 +25,36 @@ For higher level abstractions built on top of this repo, please see the followin
 
 ## Table of Contents
 
-- [Azure SDK for Embedded C](#azure-sdk-for-embedded-c)
-  - [Table of Contents](#table-of-contents)
-  - [Documentation](#documentation)
-  - [The GitHub Repository](#the-github-repository)
-    - [Services](#services)
-    - [Structure](#structure)
-    - [Main Branch](#main-branch)
-    - [Release Branches and Release Tagging](#release-branches-and-release-tagging)
-  - [Getting Started Using the SDK](#getting-started-using-the-sdk)
-    - [CMake](#cmake)
-    - [CMake Options](#cmake-options)
-    - [Consume SDK for C as Dependency with CMake](#consume-sdk-for-c-as-dependency-with-cmake)
-    - [Visual Studio Code](#visual-studio-code)
-    - [Source Files (IDE, command line, etc)](#source-files-ide-command-line-etc)
-  - [Running Samples](#running-samples)
-    - [Storage Sample](#storage-sample)
-    - [Libcurl Global Init and Global Clean Up](#libcurl-global-init-and-global-clean-up)
-    - [IoT samples](#iot-samples)
-    - [Development Environment](#development-environment)
-    - [Windows](#windows)
-      - [Visual Studio 2019](#visual-studio-2019)
-    - [Linux](#linux)
-      - [vcpkg](#vcpkg)
-      - [Debian](#debian)
-      - [Build](#build)
-    - [Mac](#mac)
-      - [vcpkg](#vcpkg-1)
-      - [Build](#build-1)
-    - [Using your own HTTP stack implementation](#using-your-own-http-stack-implementation)
-    - [Link your application with your own HTTP stack](#link-your-application-with-your-own-http-stack)
-  - [SDK Architecture](#sdk-architecture)
-  - [Contributing](#contributing)
-    - [Additional Helpful Links for Contributors](#additional-helpful-links-for-contributors)
-    - [Community](#community)
-    - [Reporting Security Issues and Security Bugs](#reporting-security-issues-and-security-bugs)
-    - [License](#license)
+- [Table of Contents](#table-of-contents)
```

Comment:
This can also be removed - self-referencing.

Reply:
these are autogenerated by an extension on save. I did revert it to have less change. Just includes the newest header for "Trademark" now